pkg/types/notification.go

```diff
@@ -23,18 +23,18 @@ import (
 	"github.com/Peripli/service-manager/pkg/util"
 )
 
-// OperationType is the notification type
-type OperationType string
+// NotificationOperation is the notification type
+type NotificationOperation string
 
 const (
 	// CREATED represents a notification type for creating a resource
-	CREATED OperationType = "CREATED"
+	CREATED NotificationOperation = "CREATED"
```

Comment:
Since we already have Notifications records on landscapes, if we're going to reuse types, we need to reuse Notification types. But there are some problems with that, like "CREATED", "MODIFIED", "DELETED" don't fit with what we understand for an operation type - what does it mean for an operation to be of type "MODIFIED"?  It's much more understandable that this is an "UPDATE" operation.

Also I'm not sure we wan't to lose this strongly typed relation that we have for Operations and Notifications.

More details on the design and task: 
https://wiki.wdf.sap.corp/wiki/display/PFS/SM+-+Asynchronous+API https://jtrack.wdf.sap.corp/browse/SVCMGR-859